### PR TITLE
[FIX] Check for valid non-node locations in CLI

### PR DESCRIPTION
**Pull request by avincigu**

### CODE REVIEW

**Review by paul-amonson — DISMISSED** [review]
Looks good to me!  :)

**Comment by paul-amonson** [comment]
> Looks good to me! :)

Except that I needs to build. ;)

**Comment by avincigu** [comment]
> > Looks good to me! :)
> 
> Except that I needs to build. ;)

I missed adding the test file

**Comment by paul-amonson** [comment]
You are still missing some code wrt the method renaming. try grep get getLocationFromHostname -- $(find . -name '*.java')

File: build.gradle [hunk 1]
```diff
@@ -612,6 +612,7 @@ project(':dai_core') {
         implementation project(':perflogging')
         implementation project(':runtime_utils')
         implementation project(':volt_init')
+        implementation project(":foreign_bus")
```
Comment (cmcheung-UCS) [inline build.gradle:615]: Should dai_core have more dependencies?

**Review by paul-amonson — APPROVED** [review]

**Review by nbwhite — APPROVED** [review]

**Pull request merged**